docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -173,21 +173,20 @@ precises the method for calculating the instantaneous velocity $$v_i(t)$$
 ## Methods
 
   Indicates the parameters related to each measurement method.
-  Six different methods `method_A` to `method_D` and `Method_I` to `Method_J` are integrated in the current
-  version of `JPSreport` and can be chosen for the analysis. The five methods `method_A` to `method_D` and `method_J` are used to analyze the steady state.
+  Four different methods `method_A` to `method_D`are integrated in the current
+  version of `JPSreport` and can be chosen for the analysis. They are used to analyze the movement of pedestrians for the steady state.
 
 | Method | measurement area | output data |
 | :---:  |     :---:        | :---:       |
 | *A* | ![Method A]({{ site.baseurl }}/images/jpsreport_Method_A.png) |  $$\langle v \rangle_{\Delta t}$$ and  $$\langle J \rangle_{\Delta t}$$ |
 | *B* | ![Method B]({{ site.baseurl }}/images/jpsreport_Method_B.png) | $$\langle v \rangle_i$$ and $$\langle \rho \rangle_i$$ |
 | *C* | ![Method C]({{ site.baseurl }}/images/jpsreport_Method_C.png) | $$\langle v \rangle_{\Delta x}$$ and $$\langle \rho \rangle_{\Delta x}$$ |
 | *D* | ![Method D]({{ site.baseurl }}/images/jpsreport_Method_D.png) | $$\langle v \rangle_v$$ and $$\langle \rho \rangle_v$$ |
-| *J* | ![Method J]({{ site.baseurl }}/images/jpsreport_Method_D.png) | $$\langle v \rangle_v$$ and $$\langle \rho \rangle_v$$ |
 
   Further information relating to each method can be found
   in [Pedestrian fundamental diagrams: Comparative analysis of experiments in different geometries](http://hdl.handle.net/2128/4898).
 
-`Method_I` can be used for time-series analysis of individual data.
+`Method_D` with `<local_IFD enabled="true"/>` can be used for time-series analysis of individual data.
```

Comment:
@JetteSchumann Didn't you mean `<global_IFD enabled="true"/>`?

Reply:
Thanks. Done!